include/qinterface.hpp

```diff
@@ -22,8 +22,10 @@
 #define bitsInByte 8
 
 #include "common/complex16simd.hpp"
+#include "common/complex16x2simd.hpp"
 
```

Comment:
These defines should really be in the complex type headers rather than here.  Probably each in their respective header, for now.

Reply:
I disagree. The type has an explicitly descriptive name. The define gives a shorthand that could be held fixed if the underlying class it referred to was changed, like using the standard library type instead. The alias is for any external code that uses the class, not the class itself. I've switched the class for the definition before, and, up to literally like a couple of instances, the alias is pretty much totally compatible with the standard library type. We could find those couple of instances where the API differs a tiny bit, again, and shore this up, but I think that's trivial if the need arises.